mgmt_cli_test.py

```diff
@@ -16,8 +16,11 @@
 import os
 import time
 import re
+import boto3
 from random import randint
 
+import libcloud.storage.types
```

Comment:
libcloud should be replaced with google native APIs.
That is since it is a 3rd party limited indirect and a risky library for compatibility.
See how we addressed it in siren-tests repository, for example by using:
`import googleapiclient.discovery
from google.oauth2 import service_account`